docs/build-walkthrough/build-process.md

```diff
@@ -3,24 +3,27 @@ title: Development Guide
 sidebar_label: Development Guide
 ---
 
```

Comment:
Would be good to have one intro sentence here.

Reply:
Updated intro sentence. I added a sentence for the other sections also. I think it looks better but let me know your thoughts.